osbuild/util/ostree.py

```diff
@@ -215,16 +216,36 @@ def parse_input_commits(commits):
     return commits["path"], data["refs"]
 
 
-def deployment_path(root: PathLike, osname: str = "", ref: str = "", serial: int = None):
-    """Return the path to a deployment given the parameters"""
+# The `deployment` arg could contain the following example values: 
+# - osname: fedora-coreos
+# - ref: 39a5ae2104301b8549fbbae3f4ec01ee689fc319ac837ec842374ee6792a1174
+# - serial: 0
+# - default: true/false
```

Comment:
We probably would want this to be a little more descriptive, kind of like the descriptions of the values in the input manifest. It can include example values, but the description is more important IMO.

Also let's not use a commit ID for the ref example (if we include an example), since the more likely input there is something like `fedora/aarch64/coreos/next` and not a commit hash.

And we'd want this function description to summarize what the function is doing.

Reply:
Would it be better to maybe include this within the triple double quotes under the function name itself?

Reply:
yeah that works for me.